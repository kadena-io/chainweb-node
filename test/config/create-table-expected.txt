- output:
  - value:
      guard:
        pred: keys-all
        keys:
        - f880a433d6e2a13a32b6169030f56245efdd8c1b8a5027e9ce98a88e886bef27
      balance: 84.01
    key: miner
    table: USER_coin_coin-table
  cmd: coinbase
- output:
  - value:
      guard:
        pred: keys-all
        keys:
        - 368820f80c324bbc7c2b0610688a7da43e39f91d118732671cd9c7500ff43cca
      balance: 999.98
    key: sender00
    table: USER_coin_coin-table
  - value:
      utModule:
        namespace: null
        name: test1
    key: test1_accounts
    table: SYS_usertables
  - value:
      guard:
        pred: keys-all
        keys:
        - 368820f80c324bbc7c2b0610688a7da43e39f91d118732671cd9c7500ff43cca
      balance: 999.98
    key: sender00
    table: USER_coin_coin-table
  - value:
      utModule:
        namespace: null
        name: test1
    key: test1_accounts
    table: SYS_usertables
  - value:
      guard:
        pred: keys-all
        keys:
        - f880a433d6e2a13a32b6169030f56245efdd8c1b8a5027e9ce98a88e886bef27
      balance: 84.02
    key: miner
    table: USER_coin_coin-table
  cmd:
    tag: Code
    contents: (create-table test1.accounts)
