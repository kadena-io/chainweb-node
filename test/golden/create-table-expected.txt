- output: oH8vcVX1AcVdO7LdS1W_86pOPk8caUeQ53xieNrk7p4
  cmd: coinbase
- output: ksiTFt9U9X42gmnYshWDjfcZS8UZ0ZO9wq3sU3Xr9A8
  cmd:
    tag: Code
    contents: (create-table test1.accounts)
